just_audio/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.9.5
```

Comment:
This part of the change log should already be in `dev` so I'm not sure why it's showing up in the diff.